src/test/ui/proc-macro/attribute-order-restricted.rs

```diff
@@ -1,14 +0,0 @@
-// aux-build:test-macros.rs
-
-#[macro_use]
-extern crate test_macros;
-
-#[identity_attr] // OK
-#[derive(Clone)]
-struct Before;
-
-#[derive(Clone)]
-#[identity_attr] //~ ERROR macro attributes must be placed before `#[derive]`
-struct After;
-
-fn main() {}
```

Comment:
Should this be kept as `check-pass`?

Reply:
Addressed below https://github.com/rust-lang/rust/pull/79078/files#diff-9e249be7ee17c294ff08ac77fc423d0aaaae6b64e26e46e02a481ad9d3d6110b